drivers/dai/intel/dmic/dmic.c

```diff
@@ -376,10 +347,6 @@ static int dai_dmic_remove(struct dai_intel_dmic *dmic)
 	uint32_t active_fifos_mask = dai_dmic_global.active_fifos_mask;
 	uint32_t pause_mask = dai_dmic_global.pause_mask;
 
-	LOG_INF("dmic_remove()");
-
-	irq_disable(dmic->irq);
```

Comment:
is the `.irq` field in `struct dai_intel_dmic` still needed after this? If not, can we remove it to make sure it isn't used by accident?

Reply:
Technicall DMIC still has irq and it could be used, also thats why I didnt removed it from DMIC dtsi

Reply:
Hrm, but, the DMIC irq is for the DMIC. It is just that the mic privacy is piggy-backing on the DMIC irq to signal a change in mic privacy.
Or to be correct: you _can_ enable the mic privacy interrupt to be delivered via DMIC instead of the generic mic privacy interrupt.